packages/time-signature/test.ts

```diff
@@ -34,6 +34,17 @@ describe("time-signature", () => {
     expect(TimeSignature.get("3+2+2/8").type).toEqual("irregular");
   });
 
+  test("get lower >= 10", () => {
```

Comment:
I guess we could change the test name...

```suggestion
  test("irrational time signatures", () => {
```